docker-compose.yml

```diff
@@ -20,21 +20,11 @@ services:
     hostname: dask-worker
     command: ["dask-worker", "tcp://scheduler:8786"]
 
-  base-notebook:
-    build:
-      context: github.com/jupyter/docker-stacks.git#master:base-notebook
-      dockerfile: Dockerfile
-      args:
-        PYTHON_VERSION: "3.8"
-    image: jupyter/base-notebook:lab
```

Comment:
Ok I think I can work with that, but for the image name I'll put it under dask/ since cross-compiled images need to pushed to a registry and we can't push to jupyter/

Reply:
Sounds good.

Reply:
Ok I've done that :) It turns out ti complicates the build process a bit because docker buildx bake runs the builds in parallel and doesn't have a way of specifying dependencies so it makes it two-step for cross-building, but regular docker-compose build continues to function as is :)